src/status_im/utils/config.cljs

```diff
@@ -24,7 +24,7 @@
 (def bootnodes-settings-enabled? (enabled? (get-config :BOOTNODES_SETTINGS_ENABLED "1")))
 (def universal-links-enabled? (enabled? (get-config :UNIVERSAL_LINK_ENABLED "1")))
 (def log-level
-  (-> (get-config :LOG_LEVEL "error")
+  (-> (get-config :LOG_LEVEL "debug")
```

Comment:
Did you mean to change the loglevel or it was used locally?

Reply:
Used locally, will fix that.